src/ramses_tx/typed_dicts.py

```diff
@@ -298,38 +332,12 @@ class AirQuality(TypedDict):
     air_quality_basis: NotRequired[str]
 
 
-class Co2Level(TypedDict):
-    co2_level: float | None
-
-
-class IndoorHumidity(TypedDict):
-    indoor_humidity: _HexToTempT
-    temperature: NotRequired[float | None]
-    dewpoint_temp: NotRequired[float | None]
-
-
 class OutdoorHumidity(TypedDict):
```

Comment:
Would you prefer to move these up with the once used in inherit? Applies to a few too.

Reply:
rebase - I have moved them myself.